web/src/app/details/DetailsPage.js

```diff
@@ -123,7 +123,7 @@ export default class DetailsPage extends React.PureComponent {
       classes,
     } = this.props
     return (
-      <Grid item container>
```

Comment:
I think the details page is rendered directly under a `Grid` container and so should be an item. `NewApp.js` may need to be cleaned up a bit though.

Reply:
I'll take a look. I didn't get any warnings or errors when I made this change, but I'll dig deeper and see if there are any parent containers above DetailsPage.

Reply:
Looks like the parent components calling `<DetailsPage />` have no reference to grids, so we should be fine here.